examples/non_cartesian_reconstruction_stack_3d.py

```diff
@@ -30,7 +30,7 @@
 
 # Loading input data
 image = get_sample_data('3d-pmri')
-image = pysap.Image(data=np.sum(image.data, axis=0))
+image = pysap.Image(data=np.sqrt(np.sum(np.abs(image.data), axis=0)))
```

Comment:
I am a bit confused here. Is it square root of sum of squares? Or just square root of absolute sum? Somehow latter does not sound right to me (the way it is coded)